### PR TITLE
[fr] Additions and cleanups to dic files

### PR DESCRIPTION
**Pull request by LucieSteib**
- cleanup in fr added
- additions and cleanup in spelling.txt fr
- acronyms and abbreviations moved to ignore.txt, duplicates removed, sorting alphabetically
- DiCaprio added to spelling_global

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt [hunk 1]
```diff
@@ -1063,4 +1063,36 @@ morinom
...
+mer.
+jeu.
+ven.
+sam.
+Plœuc
+offs
+pacemakeur
+pacemakeurs
+Virgin
+NRJ
```
Comment (tiff) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt:1081]: Please add abbreviations/acronyms to `ignore.txt`. Words in `ignore.txt` will be accepted by the speller, BUT not used to generate spelling suggestion for other misspelled words. Uncommon names and acronyms belong in there

File: languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt [hunk 2]
```diff
@@ -7624,3 +7624,4 @@ Fiat Scudo
 Citroën Jumpy
 Citroën C3
 Citroën C4
+Di Caprio
```
Comment (tiff) [inline languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt:7627]: His name is "DiCaprio" without a space (https://fr.wikipedia.org/wiki/Leonardo_DiCaprio)

Comment (LucieSteib) [reply languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt:7627]: Oh, of course! I'll change to DiCaprio then (Leonardi DiCaprio already exists, but when using only the family name, the speller complains, as expected).

**Review by GillouLT — APPROVED** [review]
A lot of the changes are fixing errors and misplacements made by myself, thanks :) Abbreviations in ignore makes total sense, and now at least we have something in this file. DiCaprio's true spelling is also a discovery on my side!

File: languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt (hunk 2 above)
Comment (LucieSteib) [reply languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt]: Just a note here to say that a rule for that is created [here](https://github.com/languagetool-org/languagetool/pull/9420)

**Pull request merged**